src/commands/openapi/upload.ts

```diff
@@ -45,30 +46,24 @@ export default class OpenAPIUploadCommand extends BaseCommand<typeof OpenAPIUplo
     useSpecVersion: Flags.boolean({
       summary: 'Use the OpenAPI `info.version` field for your ReadMe project version',
       description:
-        'If included, use the version specified in the `info.version` field in your OpenAPI definition for your ReadMe project version. This flag is mutually exclusive with `--version`.',
-      exclusive: ['version'],
-    }),
-    version: Flags.string({
-      summary: 'ReadMe project version',
-      description:
-        'Defaults to `stable` (i.e., your main project version). This flag is mutually exclusive with `--useSpecVersion`.',
-      default: 'stable',
+        'If included, use the version specified in the `info.version` field in your OpenAPI definition for your ReadMe project version. This flag is mutually exclusive with `--branch`.',
+      exclusive: ['branch'],
     }),
   };
 
   static examples = [
     {
       description: 'You can pass in a file name like so:',
-      command: '<%= config.bin %> <%= command.id %> --version=1.0.0 openapi.json',
+      command: '<%= config.bin %> <%= command.id %> --branch=1.0.0 openapi.json',
```

Comment:
added a comment below also about this.

but should we also be adding examples yet on how to upload oas files to a git branch with `--branch=1.0.0_some/branch`?